Lombiq.Hosting.MediaTheme.Deployer/Program.cs

```diff
@@ -0,0 +1,172 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Globalization;
+using System.IO.Compression;
+using System.Reflection;
+using static System.Console;
+
+namespace Lombiq.Hosting.MediaTheme.Deployer;
+
+internal static class Program
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage(
+        "Globalization",
+        "CA1303:Do not pass literals as localized parameters",
+        Justification = "It's a console application it doesn't need localization.")]
+    public static void Main(string[] args)
+    {
+        if (args.Length == 3 && bool.TryParse(args[2], out bool clearMediaFolder))
```

Comment:
I think it'd be better and more future-proof to use a library for parsing CLI. See Hastlayer.

Reply:
Can you send me a specific example? I couldn't found any.

Reply:
NVM found it.